tests/unit/scripting.tcl

```diff
@@ -993,4 +993,11 @@ start_server {tags {"scripting resp3 tests"}} {
             return redis.call('debug', 'protocol', 'false')
         } 0
     } {0}
+
+    test {test resp3 attribute protocol parsing} {
+        r eval {
+            redis.setresp(3);
+            return redis.call('debug', 'protocol', 'attrib')
```

Comment:
can we improve the test so that the script actually looks at the attributes?

Reply:
ohh, right..
well, i suppose it's a good idea to also leave a comment in the code (maybe both in the test and in the implementation)

Reply:
Fixed